### PR TITLE
BM-593: Missing Unit Tests

### PR DESCRIPTION
**Pull request by cksanders**
https://jira.digital.homeoffice.gov.uk/browse/BM-593

### CODE REVIEW

File: test/unit/controllers/Cid_EntryController.test.js [hunk 1]
```diff
@@ -233,10 +233,38 @@ describe('UNIT Cid_EntryController', () => {
...
+    afterEach(() => Centres.find.restore());
+
     var dummyMovement = [{id: 1}, {id: 2}, {id: 3}];
     it('should eventually resolve with the movements', () =>
       expect(controller.publishCentreUpdates(dummyMovement)).to.eventually.eql(dummyMovement)
     );
+
+    it('Should populate male_active_movements_in', () =>
+      controller.publishCentreUpdates()
+        .then(() => expect(Centres.find().populate).to.be.calledWith('male_active_movements_in'))
```
Comment (chrisns) [inline test/unit/controllers/Cid_EntryController.test.js]: is there a reason this different to the other tests syntax? and not `expect(populate)`?

Comment (chrisns) [inline test/unit/controllers/Cid_EntryController.test.js]: you could also (maybe?) do:
`expect(populate.getCalls()).to.eql(['male_active_movements_in', 'male_active_movements_out', 'female_active_movements_in', 'female_active_movements_in'])` ...let's talk about it :)

Comment (cksanders) [inline test/unit/controllers/Cid_EntryController.test.js]: I would prefer to divide it up into separate tests, as each is testing a different element. However, happy to change it if you prefer the cleaner and less lines implementation.

Comment (chrisns) [inline test/unit/controllers/Cid_EntryController.test.js]: nope, I'm totally happy with that rationale

**Pull request merged**